lib/coinbase/wallet.rb

```diff
@@ -459,6 +459,28 @@ def load_seed(file_path)
       "Successfully loaded seed for wallet #{id} from #{file_path}."
     end
 
+    # Creates a new webhook on the current wallet for tracking wallet activity events.
+    #
+    # @param notification_uri [String] The URI to which the webhook notifications will be sent.
+    # @param addresses [Array<String>] The list of wallet addresses to include in the event filter,
+    #   defaulting to the addresses of the current wallet.
+    # @param signature_header [String] (Optional) A header used to sign the webhook request,
+    #   defaulting to an empty string.
+    #
+    # @return [Coinbase::Webhook] The newly created webhook instance.
+    def create_webhook(notification_uri:, addresses: self.addresses.map(&:id), signature_header: '')
```

Comment:
> are we providing some sort of override to wallet addresses?
i feel they should not be able to do that... if custom group of addresses needs to be provided they can just go through webhook object
Re:
i think there could be a scenario that the wallet user only wants 1 of 3 addresses to be monitored via webhook so that they can leverage the optional addresses here

cc @cb-davidlai

Reply:
do you think we can add such parameter later?
like we discussed before, let's make it simple to use, as simple as possible would be idea
then we can extend more capabilities as we get requests